templates/zerver/help/create-user-groups.md

```diff
@@ -0,0 +1,15 @@
+# Create user groups
+
+User groups allow you to [mention](/help/mention-a-user-or-group) multiple
+users at once, [notifying](/help/pm-mention-alert-notifications) them about a message. For example, you may choose to create user groups for teams in your organization.
+
+## How to create a user group
+
+{!how-to-create-a-user-group.md!}
+
+## Related articles
+
+* [Setting up your organization](/help/getting-your-organization-started-with-zulip)
+* [User groups](/help/user-groups)
+* [Invite users to join](/help/invite-users-to-join)
+* [Getting started with Zulip](/help/getting-started-with-zulip)
```

Comment:
What's the thinking around making this a dedicated page, rather than a section within the "User groups" page?  I feel like I've been happier with pages for features of this form having a single page rather than a bunch of tiny linked pages.